molecule/default/verify.yml

```diff
@@ -70,3 +74,17 @@
       assert:
         that:
           - mail_log_result.stdout | regex_search('.*status=(bounced|sent|deferred).*')
+
+    - name: Verify Syslog output
+      shell: >
+        grep {{ vrfy_syslog}}
```

Comment:
@Chithrak07  What is it you're looking for in the syslog file?  Running `grep /filename` just hangs the terminal.

Reply:
Yup, modified this.